StRoot/StJetMaker/StJetMaker2009.h

```diff
@@ -79,7 +84,15 @@ struct StJetBranch {
   TFile* mFile;
   TTree* mTree;
 
-  ClassDef(StJetMaker2009,0);
+  // Start: bassam
+  Bool_t  mStoreOnlyDefaultVertex;
+  // End: bassam
+
+  ClassDef(StJetMaker2009,0); //zchang// bassam: this was changed from 0->1 after edits were done
```

Comment:
This added comment does not seem to correspond to the actual change... The class version is not changed from 0 to 1